plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout-block.spec.js

```diff
@@ -249,6 +238,17 @@ test.describe(
 							}
 						);
 					}
+					if (
```

Comment:
Instead of adding a new identical if block, you can update the existing one to

if (  [ newAccountEmail, newAccountEmailWithCustomPassword ].includes( response.data[ i ].billing.email ) ) {
...
}